app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -0,0 +1,64 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.location.Criteria;
```

Comment:
ERROR: Unused import: android.location.Criteria. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports